File: Python/model/Mjesanje.py
from abc import abstractmethod
from datetime import date

from model.Entitet import Entitet


class Mjesanje(Entitet):
    def __init__(self, sifra=None):
        super().__init__(sifra)
        self._stiglja = False
        self._belot = False
        self._datum_unosa = date.today()

    @abstractmethod
    def get_rezultat(self):
        pass

    @property
    def stiglja(self):
        return self._stiglja

    @stiglja.setter
    def stiglja(self, stiglja):
        self._stiglja = stiglja

    @property
    def belot(self):
        return self._belot

    @belot.setter
    def belot(self, belot):
        self._belot = belot

    @property
    def datum_unosa(self):
        return self._datum_unosa

    @belot.setter
    def datum_unosa(self, datum_unosa):
        self._datum_unosa = datum_unosa

File: Swift4/Swift4/model/Mjesanje.swift
//
//  Mjesanje.swift
//  Swift4
//
//  Created by Tomislav Jakopec on 22/02/2020.
//  Copyright © 2020 Tomislav Jakopec. All rights reserved.
//

import Foundation
class Mjesanje: Entitet{
    
    private var stiglja: Bool
    private var belot: Bool
    private var datumUnosa: Date
    
    init(sifra: Int = 0, stiglja: Bool = false, belot: Bool = false, datumUnosa: Date = Date.init()) {
        self.stiglja=stiglja
        self.belot=belot
        self.datumUnosa=datumUnosa
    }
    
    public func isStiglja()->Bool{
        return self.stiglja
    }
    
    public func setStiglja(stiglja: Bool){
        self.stiglja = stiglja
    }
    
    public func isBelot()->Bool{
        return self.stiglja
    }
    
    public func setBelot(belot: Bool){
        self.belot = belot
    }
    
    public func getDatumUnosa()->Date{
        return self.datumUnosa
    }
    
    public func setDatumUnosa(datumUnosa: Date){
        self.datumUnosa = datumUnosa
    }
    
    
}
